Update what-is-tizen "Multi device experience" (#1605)
Samsung/tizen-docs: docs/platform/what-is-tizen/overview.md

![Common infrastructure for various profiles](media/about_tizen_2.png)

> ## Multi Device Experience

Reviewer: Please use sentence style capitalization for second headings. Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization
```suggestion
## Multi device experience
```

> ## Multi device experience

Reviewer: @semun-lee, previously, I suggested sentence-style capitalization for the second heading "Multi device experience" since it was under the modified part. However, it turns out that all the other second headings on this page follow title case. To maintain consistency with other second headings now, I am requesting you to revert back the change. Since it's an urgent PR, we have decided to handle the sentence case issues later on. 

```suggestion
## Multi Device Experience
```

> Users usually use multiple smart devices in various situations. Tizen enables not only connecting those devices but also leveraging every device's capabilities for providing differentiated experience to the users.

Reviewer: An article missing before the word “differentiated”. 
```suggestion
Users usually use multiple smart devices in various situations. Tizen enables not only connecting those devices but also leveraging every device's capabilities for providing a differentiated experience to the users.
```

- **Supporting latest connectivity standards**: Tizen supports novel connectivity standards. The latest Wi-Fi standards like Wi-Fi Aware, Wi-Fi Easy Connect are supported by the platform and it adds support for new standards continuously. Matter and Thread protocols are the next generation IoT protocol for smart home devices. Especially, Tizen will support Matter protocol from the first release (1.0) of the open source matter SDK.

>   **Figure: Connectivity Standards**

Reviewer: For image captions, use sentence style capitalization.
```suggestion
  **Figure: Connectivity standards**
```

  
  <img src="media/about_tizen_7.png" alt="Connectivity Standards" width="600"/>
  

> - **Services leveraging multiple devices' capabilities**: Tizen provides easy ways of implementing services which work on multiple devices. For services on multiple devices, Tizen provides methods for discovering services among connected devices and communicating each other in efficient way. It is not bounded to Tizen-powered devices. Cross platform APIs are available for Tizen-Android inter-working.

Reviewer: I have added a preposition and an article to make it better understandable. Let me know if this seems correct to you. 
```suggestion
- **Services leveraging multiple devices' capabilities**: Tizen provides easy ways of implementing services which work on multiple devices. For services on multiple devices, Tizen provides methods for discovering services among connected devices and communicating with each other in an efficient way. It is not bounded to Tizen-powered devices. Cross platform APIs are available for Tizen-Android inter-working.
```

> For AI services, Tizen provides a framework for offloading AI inference computation to connected high-end devices. Even when a device doesn't have enough computing power to run AI models, a service can find alternative device to run the model among connected devices. By this feature every Tizen devices will be able to run AI services by leveraging peer's capability.

Reviewer: I have made some grammatical changes and restructured the last sentence to improve readability, please look into it. 
```suggestion
For AI services, Tizen provides a framework for offloading AI inference computation to connected high-end devices. Even when a device doesn't have enough computing power to run AI models, a service can find an alternative device to run the model among connected devices. By this feature, every Tizen device will be able to run AI services by leveraging the capability of peer devices.
```
